### DISCUSSION TITLE
CUSA00917  - Uncharted 4 : A Thief’s End

### DISCUSSION THREAD

**Discussion by USL5264** [GoldHEN/GoldHEN_Cheat_Repository#8] [Cheat requests]
Could cheats be added to v1.32 of this game.

These are pattern offsets found for versions of this game

Uncharted 4 v1.00
C5 FA 11 84 24 30 07 00 00 80 B8 9D 2E 00 00 00 0F 85 15 0E 00 00 80 B8 95 2E 00 00 00 0F 84 08 0E 00 00
Replace with
C5 FA 11 84 24 30 07 00 00 80 B8 9D 2E 00 00 00 0F 85 15 0E 00 00 C6 80 95 2E 00 00 01 EB 04 08 0E 00 00

Uncharted 4 v1.18
80 B8 88 2E 00 00 00 0F 85 D7 22 00 00 80 B8 79 2E 00 00 00 0F 84 CA 22 00 00
Replace with
80 B8 88 2E 00 00 00 0F 85 D7 22 00 00 C6 80 79 2E 00 00 01 EB 04 CA 22 00 00

Uncharted 4 v1.32
80 B8 88 2E 00 00 00 0F 85 D7 22 00 00 80 B8 79 2E 00 00 00 0F 84 CA 22 00 00 C5 FA 10 44 24 78
Replace with
80 B8 88 2E 00 00 00 0F 85 D7 22 00 00 C6 80 79 2E 00 00 01 EB 04 CA 22 00 00 C5 FA 10 44 24 78

Uncharted 4 v1.33
80 B8 88 2E 00 00 00 0F 85 D7 22 00 00 80 B8 79 2E 00 00 00 0F 84 CA 22 00 00 C5 FA 10 44 24 78
Replace with
80 B8 88 2E 00 00 00 0F 85 D7 22 00 00 C6 80 79 2E 00 00 01 EB 04 CA 22 00 00 C5 FA 10 44 24 78

Thanks

**Comment by kmeps4** [comment]
Hello.
Thx for the Offsets.
What will be cheat name?

**Reply by USL5264** [reply]
I thinks it's to activate the Dev Menu.

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
